lisa/sut_orchestrator/azure/platform_.py

```diff
@@ -2167,15 +2167,20 @@ def _enable_ssh_on_windows(self, node: Node) -> None:
 
         remote_node = cast(RemoteNode, node)
 
+        node_ssh_port = remote_node.connection_info[
```

Comment:
using `remote_node.connection_info[constants.ENVIRONMENTS_NODES_REMOTE_PORT]` in log.debug below was creating very long line. That's I created this variable and used it in log.debug statement below.